### PR TITLE
Version 0.15.2

### PR DESCRIPTION
**Pull request by dmdashenkov**
This PR bumps the plugin version to `0.15.2-SNAPSHOT`.

This is done in order to be able to publish the plugin into the Gradle Plugin Portal.

Also, in this PR we move scripts from `plugin/script` into `gradle` directory, as this is our new convention.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@alexander-yevsyukov, PTAL.

File: gradle/plugin.gradle [hunk 1]
```diff
@@ -37,7 +37,10 @@ project.pluginBundle {
     it.mavenCoordinates {
         groupId = 'io.spine.tools'
         artifactId = 'spine-bootstrap'
-        version = project.spineVersion
+        // Corresponds to the latest `web` version.
+        // Gradle Plugin portal does not allow to override published artifacts.
+        // Should be synchronized with the framework version as soon as 1.0.0-SNAPSHOT era is over.
+        version = '0.15.2-SNAPSHOT'
```
Comment (armiol) [inline gradle/plugin.gradle]: Can we then use `0.15.2` with no `SNAPSHOT`?

**Review by alexander-yevsyukov — CHANGES_REQUESTED** [review]
Please make it `0.15.2`.

**Comment by dmdashenkov** [comment]
@alexander-yevsyukov, PTAL again.

**Review by alexander-yevsyukov — APPROVED** [review]
LGTM

**Pull request merged**